applications/GeoMechanicsApplication/tests/cpp_tests/test_U_Pw_normal_lysmer_absorbing_condition.cpp

```diff
@@ -0,0 +1,205 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Aron Noordam
+//
+
+#include <string>
+
+// Project includes
+#include "custom_elements/U_Pw_small_strain_element.hpp"
+#include "custom_elements/plane_strain_stress_state.h"
+
+#include "geo_mechanics_fast_suite.h"
+
+namespace Kratos::Testing
+{
+
+class MockElement : public UPwSmallStrainElement<2, 4>
+{
+public:
+    MockElement() : UPwSmallStrainElement<2, 4>() {}
+
+    MockElement(IndexType NewId, GeometryType::Pointer pGeometry, PropertiesType::Pointer pProperties)
+        : UPwSmallStrainElement(NewId, pGeometry, pProperties, std::make_unique<PlaneStrainStressState>())
+    {
+    }
+
+    void CalculateOnIntegrationPoints(const Variable<double>& rVariable,
+                                      std::vector<double>&    rOutput,
+                                      const ProcessInfo&      rCurrentProcessInfo) override
+    {
+        rOutput.resize(4);
+
+        if (rVariable == SHEAR_STIFFNESS) {
+            double shear_stiffness = 5.0;
+            rOutput[0]             = shear_stiffness;
+            rOutput[1]             = shear_stiffness;
+            rOutput[2]             = shear_stiffness;
+            rOutput[3]             = shear_stiffness;
```

Comment:
I did not see this in the previous round, but the following could be a cleaner way of initializing this vector, but I leave it up to you if you want to take it up.
```suggestion
            rOutput = ScalarVector(4, shear_stiffness)
```

The same holds for the other variables

Reply:
but rOutput is an std::vector and ScalarVector is a boost vector.  I dont think this works

Reply:
Ah, you're correct, then you could use the vector constructor (which works in a similar way): 
`rOutput = std::vector<double>(4, shear_stiffness)`